git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,829 @@
+</div>
+
+
+
+### Quiz: `git init`
+
+What command Dracula enter in the blank in order to create a new Git repository?
+
+```console
+$ cd ~/Desktop      # return to the Desktop directory
+$ mkdir planets     # make the planets directory
+$ cd planets        # go into the newly created planets directory
+$ __________        # make the planets directory a Git repository
+
+```
+[[git init]]
+***
+<div class = "answer">
+By initializing the Git repository inside the `planets` directory with the `git init` command, Git will track every file inside `planets`.
+</div>
+***
+
+
+Along with tracking information about planets (the project we have already created),
+Dracula would also like to track information about moons.
+Consider this possible scenario: Dracula creates a `moons` project inside his `planets`
+project with the following sequence of commands.  Here we've added comments, the text following the `#`, explaining what each command does.
+
+```console
+$ cd ~/Desktop   # return to Desktop directory
+$ cd planets     # go into planets directory, which is already a Git repository
+$ ls -a          # ensure the .git subdirectory is still present in the planets directory
+$ mkdir moons    # make a subdirectory planets/moons
+$ cd moons       # go into moons subdirectory
+$ git init       # make the moons subdirectory a Git repository
+$ ls -a          # ensure the .git subdirectory is present indicating we have created a new Git repository
+```
+
+Is the `git init` command, run inside the `moons` subdirectory, required for
+tracking files stored in the `moons` subdirectory?
+
+[( )] Yes, running `git init` inside the `moons` subdirectory is necessary.
+[( )] It is not necessary to run `git init` inside `moons` but it can't hurt.
+[(X)] No, `moons` was already tracked as a subdirectory of `planets` and running `git init` inside `moons` could interfere with version control.
+***
+<div class = "answer">
+No. Dracula does not need to make the `moons` subdirectory a Git repository
+because the `planets` repository will track all files, sub-directories, and
+subdirectory files under the `planets` directory.  Thus, in order to track
+all information about moons, Dracula only needed to add the `moons` subdirectory
+to the `planets` directory.
+
+Additionally, Git repositories can interfere with each other if they are "nested":
+the outer repository will try to version-control
+the inner repository. Therefore, it's best to create each new Git
+repository in a separate directory. To be sure that there is no conflicting
+repository in the directory, check the output of `git status`. If it looks
+like the following, you are good to go to create a new repository as shown
+above:
+
+```console
+$ git status
+
+fatal:
+Not a git repository (or any of the parent directories):
+.git
+```
+ </div>
+ ***
+
+### Fixing a nested `init`
+
+What if you forgot to check `git status` and accidentally created a repository nested in another repository?
```

Comment:
I don't think you've introduced `git status` yet within this module. It might be worth adding a hint or link to a brief explanation of what `git status` does and why it would have helped prevent the accidental creation of a nested `init`.